vllm/worker/model_runner.py

```diff
@@ -175,7 +187,7 @@ def _prepare_prompt(
             # NOTE(woosuk): Here we assume that the first token in the prompt
             # is always the first token in the sequence.
             input_positions.extend(
-                list(range(computed_len, computed_len + len(prompt_tokens))))
+                list(range(computed_len, computed_len + prefill_end)))
```

Comment:
Please validate the correctness for this, i suppose in general range(prefill_start, prefill_end) should work?

Reply:
I changed it to this way; 

if prefix caching is not used, computed_len = prefill_start (which is 0 all the time in this PR, and can be changed in the future). 

And it is range(computed_len, prefill_end).